spec-conformance-tests/S05_structured_values/tests/lists/arrays/array_inherent_type_violation.bal

```diff
@@ -0,0 +1,140 @@
+// Copyright (c) 2019 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/test;
+import utils;
+
+const string INHERENT_TYPE_VIOLATION_REASON = "{ballerina}InherentTypeViolation";
+const string INVALID_REASON_ON_INHERENT_TYPE_VIOLATING_ARRAY_INSERTION_FAILURE_MESSAGE =
+                                            "invalid reason on inherent type violating array insertion";
+
+// The inherent type of a list value determines a type Ti for a member with index i.
+// The runtime system will enforce a constraint that a value written to index i will
+// belong to type Ti. Note that the constraint is not merely that the value looks
+// like Ti.
+
+@test:Config {}
+function testBasicTypeArrayInherentTypeViolation() {
+    int[] intArray = [1, 2];
+    any[] anyArray = intArray;
+    utils:assertPanic(function() { anyArray[intArray.length() - 1] = "not an int"; },
+                      INHERENT_TYPE_VIOLATION_REASON,
```

Comment:
Can be moved to the above line.

Reply:
We believe these line breaks make it more readable, specially with the `utils:assertPanic` function where we specify the function itself inline and then two constants.

```ballerina
    utils:assertPanic(function() { anyArray[0] = stringOrIntArray; }, INHERENT_TYPE_VIOLATION_REASON,
                      INVALID_REASON_ON_INHERENT_TYPE_VIOLATING_ARRAY_INSERTION_FAILURE_MESSAGE);
```

Applicable for other occurrences too.

WDYT?

Reply:
The current format looks better. +1 for keeping it.